DependencyInjection/Configuration.php

```diff
@@ -57,7 +62,7 @@ public function getConfigTreeBuilder()
                 ->arrayNode('param_fetcher_listener')
                     ->beforeNormalization()
                         ->ifString()
-                        ->then(function ($v) { return array('enabled' => in_array($v, array('force', 'true')), 'force' => 'force' === $v); })
+                        ->then(function ($v) { return ['enabled' => in_array($v, ['force', 'true']), 'force' => 'force' === $v]; })
```

Comment:
these changes are weird though.

@GrahamCampbell is the ``symfony`` preset enforcing the short array syntax on StyleCI ? This would be weird as Symfony itself explicitly chose not to use it.